src/TensorFlow.jl

```diff
@@ -99,6 +99,8 @@ while_loop,
 get_tensor_by_name,
 as_default,
 @tf,
+withname,
+tf_while,
```

Comment:
I'ld rather leave these un-exported too.
I was having namespace issues inside the `@tf` macro.
When the `@tf` macro was run from the `.nn` submodule etc.
I'll give fixing that another shot.